source/class/qx/tool/compiler/resources/ImageLoader.js

```diff
@@ -69,7 +86,7 @@ qx.Class.define("qx.tool.compiler.resources.ImageLoader", {
         fileInfo.width = dimensions.width;
         fileInfo.height = dimensions.height;
       } catch (ex) {
-        log.warn("Cannot get image size of " + filename + ": " + ex);
+        //log.warn("Cannot get image size of " + filename + ": " + ex);
```

Comment:
This should either warn (be uncommented) or be elided. If no warning is needed, then just a comment in the `catch` that says `// couldn't get image size` or something would be adequate.

Reply:
oops, very much so